elements/pfe-card/src/pfe-card.scss

```diff
@@ -13,6 +13,8 @@
   display: flex;
   flex-direction: column;
   justify-items: flex-start;
+  // This allows the card to fill it's container if necessary
+  align-self: stretch;
```

Comment:
@starryeyez024 This will fix the card filling the space

Reply:
ack thanks, removed from my PR https://github.com/patternfly/patternfly-elements/pull/336/commits/1ce9b3d63ab5fdf3abd70d556ded2f45b357dc2c